tensorflow_addons/image/transform_test.py

```diff
@@ -22,9 +22,17 @@
 import tensorflow as tf
 
 from tensorflow.python.ops import gradient_checker
+from tensorflow.python.framework import dtypes
+from tensorflow.python.ops import array_ops
+from tensorflow.python.ops import math_ops
+from tensorflow.python.framework import constant_op
+from tensorflow.python.ops import random_ops
+
 from tensorflow_addons.image import transform_ops
+from tensorflow_addons.image import transform_ops as image_ops  # symlink allows direct source copy from tf.113
```

Comment:
Why we need it? Can we remove the alias?

Reply:
This allowed us to copy-paste the old test code as-is, which has some value in preserving the code provenance to the old repo and commit history.  Up to you.